src/gp/lti_sde.jl

```diff
@@ -246,7 +246,29 @@ Zygote.@adjoint function stationary_distribution(k::Matern52Kernel, storage_type
     return stationary_distribution(k, storage_type), Δ->(nothing, nothing)
 end
 
+# constant
 
+function TemporalGPs.to_sde(k::ConstantKernel{T}, ::SArrayStorage{T}) where {T<:Real}
+    F = SMatrix{1, 1, T}(0)
+    q = convert(T, 0)
+    H = SVector{1, T}(1)
+    return F, q, H
+end
+
+function TemporalGPs.stationary_distribution(k::ConstantKernel{T}, ::SArrayStorage{T}) where {T<:Real}
+    return TemporalGPs.Gaussian(
+        SVector{1, T}(0),
+        SMatrix{1, 1, T}(1),
```

Comment:
I think this probably needs to be `k.c[1]` rather than `1` -- the `ConstantKernel` kernel has a parameter for some reason (I'm not sure why we still have it, probably convenience). This will mean that the `@adjoint` for stationary distribution will have to be changed -- possibly just not implementing anything will be fine.

Reply:
> This is looking great -- I think it just needs generalising to allow for the constant in `ConstantKernel` not being `1`.

right! as I understand, k.c has the same effect as the scaling factor σ², i.e. the kernel variance and is therefore somewhat redundant. Nevertheless, we should take care of it

Reply:
> I think this probably needs to be `k.c[1]` rather than `1`
I agree. That would be the initial state variance

Reply:
> This will mean that the `@adjoint` for stationary distribution will have to be changed -- possibly just not implementing anything will be fine.

you mean, we should drop the adjoint
```julia
Zygote.@adjoint function TemporalGPs.stationary_distribution(k::ConstantKernel, storage_type)
    return TemporalGPs.stationary_distribution(k, storage_type), Δ->(nothing, nothing)
end
```
altogether?

In this case, we would have to define an adjoint for the constructor of a static array. Otherwise, I would get the error message:
> Need an adjoint for constructor SMatrix{1, 1, Float64, 1}. Gradient is of type SMatrix{1, 1, Float64, 1}

I guess, something like 
```julia
Zygote.@adjoint SMatrix{S,S,T}(args...) where {S,T} = SMatrix{S,S,T}(args...), Δ -> nothing
```
does it?

Reply:
Yeah, something like this, but you'll need to make sure that the value is propagated back. I'm a bit surprised that [this rule](https://github.com/JuliaGaussianProcesses/TemporalGPs.jl/blob/d56d1cde388e65aa1c3c738020ef7c0ebb4690c2/src/util/zygote_rules.jl#L41) doesn't handle the problem.

I'll investigate locally and get back to you.

Reply:
Could you try replacing the rule I linked above with:
```julia
function Zygote._pullback(::AContext, ::Type{<:SMatrix{1, 1}}, a)
    SMatrix_pullback(::Nothing) = nothing
    SMatrix_pullback(Δ::AbstractMatrix) = (nothing, first(Δ), )
    return SMatrix{1, 1}(a), SMatrix_pullback
end
```
and see if it fixes the problem? If so, please just add this in to the PR :)

Reply:
thats right and it works :)